src/components/CohortBuilder/ParticipantsTableView/SaveSetModal.tsx

```diff
@@ -182,13 +235,17 @@ const SaveSetModal: FunctionComponent<Props> = (props) => {
         </Form.Item>,
       ]}
     >
-      {loadingDefaultTagName ? (
+      {SaveSetActionsTypes.CREATE === saveSetActionType && loadingDefaultTagName ? (
```

Comment:
Are you sure it's not a || vs &&

Reply:
yes its ok. there can only be a spin in the case of a create of Save set